src/transformers/generation_utils.py

```diff
@@ -1179,6 +1195,7 @@ def generate(
             repetition_penalty=repetition_penalty,
             no_repeat_ngram_size=no_repeat_ngram_size,
             encoder_no_repeat_ngram_size=encoder_no_repeat_ngram_size,
+            input_ids=input_ids,
```

Comment:
Can we replace all occurences af `input_ids.shape[-1]` with a variable called `input_ids_seq_length` and pass this one here instead?